ownerdelete.user.js

```diff
@@ -217,10 +240,10 @@ dry.once("load", () => {
         {white: true} :
         {green: true});
       file.dom.setTags(tags);
+      pool.schedule(getInfo, file);
```

Comment:
I personally prefer pool.wrap (aka bind a function to always use  the pool) instead of explicitly scheduling every call.

Reply:
I use explicit calls because when I wrap the function, I don't have access to the getters, which are required later.